### PR TITLE
Add support for tar-mode

### PR DESCRIPTION
**Pull request by 0x28**
This PR adds support for the builtin `tar-mode`. I tried to keep it similar to the existing `arc-mode` configuration.

### CODE REVIEW

File: modes/tar-mode/evil-collection-tar-mode.el [hunk 1]
```diff
@@ -0,0 +1,81 @@
+;;; evil-collection-tar-mode.el --- Evil bindings for tar-mode. -*- lexical-binding: t -*-
+
+;; Copyright (C) 2019 James Nguyen
```
Comment (Ambrevar) [inline modes/tar-mode/evil-collection-tar-mode.el]: Copyright should be assigned to you ;)

**Comment by Ambrevar** [comment]
Great contribution, thanks!  Can you just fix the copyright header?

**Comment by 0x28** [comment]
Forgot about that :)

I hope it's ok now.

**Comment by Ambrevar** [comment]
Thanks!

**Pull request merged**